idea/idea-fir/tests/org/jetbrains/kotlin/idea/quickfix/AbstractHighLevelQuickFixTest.kt

```diff
@@ -7,20 +7,90 @@ package org.jetbrains.kotlin.idea.quickfix
 
 import com.intellij.codeInsight.intention.IntentionAction
 import com.intellij.codeInspection.InspectionProfileEntry
+import com.intellij.openapi.util.io.FileUtil
 import org.jetbrains.kotlin.test.uitls.IgnoreTests
+import java.io.File
 import java.nio.file.Paths
 
 
 abstract class AbstractHighLevelQuickFixTest : AbstractQuickFixTest() {
+    companion object {
+        private val isTeamCityBuild: Boolean = System.getProperty("TEAMCITY_VERSION") != null
+    }
+
     override fun doTest(beforeFileName: String) {
         IgnoreTests.runTestIfEnabledByFileDirective(
             Paths.get(beforeFileName),
-            enableTestDirective = IgnoreTests.DIRECTIVES.FIR_COMPARISON_MUTLTILINE_COMMENT,
+            enableTestDirective = IgnoreTests.DIRECTIVES.FIR_COMPARISON_MULTILINE_COMMENT,
             directivePosition = IgnoreTests.DirectivePosition.LAST_LINE_IN_FILE,
             additionalFilesExtensions = arrayOf("after")
         ) {
             super.doTest(beforeFileName)
+            // getFir/OriginalAfterFile expect just the filename (no directory), but beforeFileName has the directory
+            val beforeFileNameOnly = File(beforeFileName).name
+            val firAfterFile = getFirAfterFile(beforeFileNameOnly)
+            val originalAfterFile = getOriginalAfterFile(beforeFileNameOnly)
+            if (firAfterFile.exists() && firAfterFile.readText().trim() == originalAfterFile.readText().trim()) {
+                // The FIR after file is identical with the original after file. Remove the FIR after file and mark "FIR_IDENTICAL" in the original files.
+                // Note: FIR_IDENTICAL cannot be at the start of the file because the test uses ActionHint which parses the first line to get
+                // the action to perform (or check that they are not available).
+                firAfterFile.delete()
```

Comment:
As per the message below, we won't delete the after file if `isTeamCityBuild`?

Reply:
Thanks for pointing that out! I decided to just delete the whole FIR_IDENTICAL logic following Ilya's suggestion